src/Articles.jsx

```diff
@@ -0,0 +1,24 @@
+import { useState, useEffect } from "react";
+import ArticleCard from "./ArticleCard";
+
+export default function Articles() {
+  const [articlesList, setArticlesList] = useState([]);
+
+  useEffect(() => {
+    fetch("https://rafaels-nc-news.onrender.com/api/articles")
+      .then((response) => {
+        return response.json();
+      })
+      .then(({ articles }) => {
+        setArticlesList(articles);
+      });
```

Comment:
I'd encourage that you setup an `isLoading` pattern so the user knows data is being gathered. At first the screen was blank for ages and I thought it had broke! [until the articles finally loaded]

Reply:
done